src/components/Form/Form.jsx

```diff
@@ -36,7 +36,10 @@ const Form = ({ children, onSubmit, formData = e => e, FormButtonTitle = 'Confir
       >
         <FormContext.Consumer>{children}</FormContext.Consumer>
       </FormContext.Provider>
-      <FormButton type="submit">{FormButtonTitle}</FormButton>
+      <ButtonContainer>
+      { FormButtonSecondaryTitle != null ? <FormButtonSecondary type="submit">{FormButtonSecondaryTitle}</FormButtonSecondary> : null}
```

Comment:
As doesn't exists an "else" action, you can to use:

```suggestion
      { FormButtonSecondaryTitle && <FormButtonSecondary type="submit">{FormButtonSecondaryTitle}</FormButtonSecondary> }
```

Reply:
ready